### ISSUE TITLE
All components FOUC the page regardless of resource load timing or page caching

### ISSUE THREAD

**Issue by csuwildcat** [mozbrick/brick#196]
Not sure if this is already something you're looking at with these new non-X-tag components, but I noticed all the new versions of these components flash a broken state regardless of when they are loaded, or if the page is cached.

**Comment by potch** [comment]
Yep- it's an unfortunate side effect of HTMLImports not blocking page load. Polymer offers an [unresolved] attribute for the purposes of obscuring the FOUC, which we will likely use and document as a best practice. Most complex webapps have a splash screen, and perhaps even a `<splash-screen>` is something developers may want.
